src/explore-education-statistics-common/src/modules/table-tool/components/TimePeriodDataTable.tsx

```diff
@@ -310,14 +311,15 @@ const TimePeriodDataTableInternal = forwardRef<HTMLElement, Props>(
 
     const rows = filteredCartesian.map(row => row.map(cell => cell.text));
 
+    const id = dataBlockId
```

Comment:
One minor thing - would just rename this to `captionId` to make it a little more explicit what it refers to